hedgehog-example/hedgehog-example.cabal

```
@@ -53,6 +53,7 @@ library
       hedgehog
     , hedgehog-quickcheck
     , base                            >= 3          && < 5
+    , barbies
```

```suggestion
    , barbies                         >= 2.0        && < 2.1
```
(or whatever lower bound works)